internal/pkg/cli/app_show.go

```diff
@@ -96,21 +102,14 @@ func (o *showAppOpts) Execute() error {
 		// If there are no local applications in the workspace, we exit without error.
 		return nil
 	}
-	err := o.initDescriber(o)
+	err := o.initDescriber(o, o.shouldOutputResources)
 	if err != nil {
 		return err
 	}
 	var app *describe.WebAppDesc
-	if o.shouldOutputResources {
-		app, err = o.describer.Describe(describe.WithAppResources())
-		if err != nil {
-			return fmt.Errorf("describe application %s: %w", o.appName, err)
-		}
-	} else {
-		app, err = o.describer.Describe(describe.WithNoAppResources())
-		if err != nil {
-			return fmt.Errorf("describe application %s: %w", o.appName, err)
-		}
+	app, err = o.describer.Describe()
```

Comment:
can we remove the above `var` with a short declaration `app, err := o.describer.Describe()`

Reply:
oops that was a mistake